AdminWebsite/AdminWebsite/Controllers/ReferenceDataController.cs

```diff
@@ -60,13 +60,12 @@ public async Task<ActionResult<IList<HearingTypeResponse>>> GetHearingTypes([Fro
                 Code = hearingType.Code
             } )).ToList();
             
-            if (_featureToggles.UseV2Api())
-                result.AddRange(caseTypes.Where(ct => !ct.HearingTypes.Any())
-                    .Select(caseType => new HearingTypeResponse
-                    {
-                        Group = caseType.Name,
-                        ServiceId = caseType.ServiceId
-                    }));
+            result.AddRange(caseTypes.Where(ct => !ct.HearingTypes.Any())
```

Comment:
The ref data toggle needs to also be removed

Reply:
done